components/BattleScreen/PlayerState/PlayerState.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const PlayerStateSection = styled.section`
+	position: absolute;
+	top: 165px;
+	right: 0px;
+	background-image: url("/sprites/statusbar-player3.png");
+	width: 200px;
+	height: 68px;
+	background-repeat: no-repeat;
+	background-size: contain;
+	z-index: 1;
+`;
+
+const PokemonName = styled.p`
+	position: absolute;
+	top: 5px;
+	left: 30px;
+	z-index: 1;
+	text-shadow: 1px 1px 0px #b7b7b7;
+	font-size: 20px;
+	text-transform: uppercase;
+	margin: 0;
+	padding: 0;
+`;
+
+const PokemonLevel = styled.span`
+	position: absolute;
+	top: 5px;
+	right: 29px;
+	z-index: 10;
+	text-shadow: 1px 1px 0px #b7b7b7;
+	font-size: 20px;
+	margin: 0;
+	padding: 0;
+`;
+
+const PokemonLevelTitel = styled.span`
+	position: absolute;
+	top: 5px;
+	right: 50px;
+	z-index: 10;
+	text-shadow: 1px 1px 0px #b7b7b7;
+	font-size: 20px;
+	margin: 0;
+	padding: 0;
+`;
+
+const PokemonHealth = styled.span`
+	position: absolute;
+	top: 40px;
+	right: 55px;
+	z-index: 11;
+	text-shadow: 1px 1px 0px #b7b7b7;
+	font-size: 15px;
+	margin: 0;
+	padding: 0;
+`;
+
+const PokemonHealthBarContainer = styled.progress`
+	position: absolute;
+	height: 6px;
+	width: 89px;
+	left: 88px;
+	top: 31px;
```

Comment:
I really hope that these fixed values won't create any trouble regarding responsiveness!

Reply:
Yes, this is still a bigger issue. Not sure how to solve this perfectly.